### PR TITLE
Robolectric fails to load resources from depending modules

### PR DESCRIPTION
**Pull request by felipecsl**
Still not sure whether Robolectric or Buck issue or if there is a
workaround we could do in the test runner, but putting this up here for
now just for documentation/discussion/feedback.
Added a failing test that demostrates the issue.

### CODE REVIEW

File: libraries/common/src/test/java/com/uber/okbuck/example/common/ExampleUnitTest.java [hunk 1]
```diff
@@ -25,5 +24,8 @@ public void addition_isCorrect() throws Exception {
     public void resource_loading() {
         Context context = (Context) RuntimeEnvironment.application;
         assertEquals(context.getResources().getString(R.string.app_name), "Common");
+        assertEquals(context.getResources().getColor(
+            com.uber.okbuck.example.parcelable.R.color.fooColor), 0xcccccc);
```
Comment (kageiit) [inline libraries/common/src/test/java/com/uber/okbuck/example/common/ExampleUnitTest.java]: This is probably not the right example. This specific problem goes away when you use the Right R class here.

Can you modify this change to repro your case i.e have another module refer to the resource in its main code and this module invokes that piece of code in it's test?

Comment (felipecsl) [inline libraries/common/src/test/java/com/uber/okbuck/example/common/ExampleUnitTest.java]: Yes, you're right. Good point.

File: libraries/parcelable/src/main/java/com/uber/okbuck/example/common/Person.java [hunk 2]
```diff
@@ -1,8 +1,11 @@
 package com.uber.okbuck.example.common;
 
+import android.content.Context;
 import android.os.Parcel;
 import android.os.Parcelable;
 
+import com.uber.okbuck.example.parcelable.R;
```
Comment (kageiit) [inline libraries/parcelable/src/main/java/com/uber/okbuck/example/common/Person.java]: does this error happen if you use `com.uber.okbuck.example.common.R` ?

Comment (felipecsl) [inline libraries/parcelable/src/main/java/com/uber/okbuck/example/common/Person.java]: @kageiit test updated. `com.uber.okbuck.example.common.R` works, but `com.uber.okbuck.example.parcelable.R` does not

Comment (felipecsl) [inline libraries/parcelable/src/main/java/com/uber/okbuck/example/common/Person.java]: That's the one I'm using, `com.uber.okbuck.example.parcelable.R` is the merged R for this library

**Comment by kageiit** [comment]
@seanabraham @felipecsl This seems to be working if you use the default robolectric test runner that comes as part of robolectric (which handles the buck build system correctly now). I rebased your branch to the latest okbuck master on the felipecsl-robolectric-resources branch. Feel free to check the same and  I can close this ticker

**Pull request closed without merging**